hasura/metadata/cron_triggers.yaml

```diff
@@ -110,3 +110,12 @@
     - name: verification_key
       value_from_env: HASURA_EVENT_SECRET
   comment: syncs cosouls in db and sets pgive on chain
+- name: updateMagicEmails
+  webhook: '{{HASURA_API_BASE_URL}}/cron/updateMagicEmails'
+  schedule: 0 * * * *
```

Comment:
Since we will batch 20 at a time, just change this to run every minute * * * * *